modules/project-docs/partials/attributes.adoc

```diff
@@ -2,3 +2,4 @@
 :ruby-current-version: 3.3.0
 :name-sdk: Ruby SDK
 :version-server: 7.1
+:version-common: 7.1.2
```

Comment:
Most times I expect `version-server` and `version-common` to be the same, but in instances like these where we have slight differences in versions I think this offers a more flexible approach.